src/cache/cache.rs

```diff
@@ -200,7 +200,8 @@ impl CacheWrite {
         pool.spawn_blocking(move || {
             let mut entry = CacheWrite::new();
             for (key, path) in objects {
-                let mut f = fs::File::open(&path)?;
+                let mut f = fs::File::open(&path)
```

Comment:
Since we use `fs-err` it's not necessary to annotate the error its already included